### ISSUE TITLE
Author fields display and index

### ISSUE THREAD

**Issue by banukutlu** [psu-libraries/psulib_blacklight#185]
See https://github.com/psu-libraries/psulib_blacklight/wiki/Author-Creator-Fields-Indexing-and-Display

**Comment by cdmo** [comment]
@ruthtillman:

> Author display 100aqbcdkj including vernacular. On item pages, this should link to a search for that author's exact name. ([src](https://github.com/psu-libraries/psulib_blacklight/wiki/Author-Creator-Fields-Indexing-and-Display))

I just want to confirm, search? Or a [faceted browse](https://librarysearch.temple.edu/catalog/991023689319703811)? Or [both](https://catalog.princeton.edu/catalog/1797022)?

**Comment by ruthtillman** [comment]
@cdmo I was thinking search, e.g. ?f[author_s][]=Bryson%2C+Bill If it makes more sense with our code to link to faceted browse, though, I don't have a particular functional objection... I'm just not as sure if the 7xx will be in faceted browse?

**Comment by cdmo** [comment]
Currently we have `100abcdqj:110abcdj:111ancdj:700abcdqj:710abcdj:711ancdj` in our `all_authors_facet_sim`, so, is that adequate you think?

**Comment by ruthtillman** [comment]
Should be ok then yeah.

**Comment by cdmo** [comment]
@ruthtillman 

Okay, for vernacular authors like [是枝裕和, 1962-](https://blackcat01qa.libraries.psu.edu/catalog/2788022), you wrote that we should "include vernaculars" in display. I'm wondering how to include them exactly. Currently [in the CAT we just put them on the same line with a space for a separator](https://cat.libraries.psu.edu/uhtbin/cgisirsi/x/0/0/57/5/0?searchdata1=2788022%7BCKEY%7D&searchfield1=GENERAL%5ESUBJECT%5EGENERAL%5E%5E&user_id=WEBSERVER) (it appears).

I kind of assume that we'd rather show the vernacular name first followed by the "latinized" version, perhaps in parenthesis? But, I am only looking at one case here, I don't know how quickly this would breakdown/not work.

**Comment by ruthtillman** [comment]
@cdmo I think one challenge is that we may not always have the vernacular author's name, so it would make more sense to link to the Latinized version for further search. An example might be someone putting in Tolstoy's name in Cyrillic, but not every record would have that. That kind of data situation would drive design decisions, although there's nothing wrong with putting the linked form in the parentheses. It would just make the logic more complex.  The "just a space" is definitely too little differentiation.

**Comment by ruthtillman** [comment]
It looks like the most common practice is to list the vernacular name directly under the other name. They may or may not both be linked to a search for that name (again, I'd be ok limiting it to the Latin). This works fine for 1xx fields, for which only one exists. This may be more challenging for 7xx fields.

**Comment by cdmo** [comment]
Cool, thanks @ruthtillman

**Comment by cdmo** [comment]
Hi @ruthtillman - so @banukutlu caught an issue in the review of the PR for this work. The facet field has different mappings then the individual fields for author, corporate author, meeting author and additional author. See:

facet author field mapped marc:
`100abcdqj:110abcdj:111ancdj:700abcdqj:710abcdj:711ancdj`

person:
`100aqbcdkj`

corp:
`110abcdfgklnj`

meeting:
`111abcdfgklnpqj`

addl:
`700aqbcdjk:710abcdfgjkln:711abcdfgjklnpq`

So you can see the display fields have more subfields and a slightly different order.

I didn't catch this in my physical/manual testing of linked author statements, but it's reasonable to assume that there could be a value mis-match leading to an empty result set.

An easy fix would be to just change the facet field to be the same as the display, but I want to make sure that's sensible! :) thanks

**Comment by ruthtillman** [comment]
yes, that should work. Reviewed subfields and I think we  can make that change.

**Comment by cdmo** [comment]
Thanks!

**Comment by cdmo** [comment]
Thanks!
